tests/unit/fork_test.cpp

```diff
@@ -0,0 +1,52 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#include <unistd.h>
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+using namespace testing;
+
+extern "C" {
+   #include <tilck/kernel/process.h>
+   #include "mocking.h"
```

Comment:
**FIXIT** mocking.h is a C++ header, designed to be used for the C++ tests, so it shouldn't be wrapped by extern "C". Put it after `#include <gmock/gmock.h>`

Reply:
Oh, ok. My bad